src/app/auth/login/login.component.html

```diff
@@ -0,0 +1,28 @@
+<body class="text-center">
```

Comment:
Shouldn't we use a div here?? Considering the entire app is rendered within the `body` tag of `index.html`. Feels like a body tag within a body tag

Reply:
yep should be a div